### PR TITLE
[NIS-13912] implemented entities, fixed drawio scheme

### PR DESCRIPTION
**Pull request by westcrime**

### CODE REVIEW

File: Services/Services.Domain/Entities/Doctor.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        public string LastName { get; set; }
+
+        [Required]
+        public Guid SpecializationId { get; set; }
+
+        [ForeignKey("SpecializationId")]
+        public Specialization Specialization { get; set; }
+
+        [Required]
+        public string Status { get; set; }
```
[the remark this answers is not in the document]
Comment (westcrime) [reply Services/Services.Domain/Entities/Doctor.cs:31]: Done

File: Services/Services.Domain/Enums/DoctorStatus.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Services.Domain.Enums
+{
+    public enum DoctorStatus
+    {
+        AtWork,
```
Comment (Glipchik) [inline Services/Services.Domain/Enums/DoctorStatus.cs:11]: There is Microsoft recommendation to always add a default value to enums like None

Comment (Glipchik) [reply Services/Services.Domain/Enums/DoctorStatus.cs:11]: https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/enum

**Review by Glipchik — CHANGES_REQUESTED** [review]

File: Services/Services.Domain/Enums/DoctorStatus.cs (hunk 2 above)
Comment (westcrime) [reply Services/Services.Domain/Enums/DoctorStatus.cs:12]: Fixed

**Review by Glipchik — APPROVED** [review]

**Pull request merged**